docs/reference/commandline/swarm_leave.md

```diff
@@ -13,16 +13,23 @@ parent = "smn_cli"
 ```markdown
 Usage:  docker swarm leave [OPTIONS]
 
-Leave a swarm
+Leave a swarm. If the node is a manager, it is recommended to
+demote it to a worker before removing it from the swarm.
 
 Options:
-      --force   Force leave ignoring warnings.
+      --force   Force a node to leave the swarm, ignoring warnings.
       --help    Print usage
 ```
 
-This command causes the node to leave the swarm.
+This command causes the node to leave the swarm. The `--force` option allows a manager to
+leave the swarm, but this does not reconfigure the swarm to cleanly remove the node as a
+manager. This can put the swarm into a bad state where it no longer has enough managers to
+obtain a quorum. The safe way to remove a manager from a swarm is to demote it to a worker
+and then direct it to leave the quorum without using `--force`. Only use `--force` in
+situations where the swarm will no longer be used after the manager leaves, such as in
+a single-node swarm.
 
-On a manager node:
+Consider the following swarm, as seen from the manager:
```

Comment:
I noticed that the node named `worker1` below is actually a manager (it has "manager status" listed as "Reachable"). Can you please remove the "Reachable" to make its appearance consistent with a worker?